providers/ldap/ldap_client.go

```diff
@@ -826,3 +837,20 @@ func (l *LClient) userRecord(search *ldap.SearchRequest, lConn *ldap.Conn) (mode
 	token.AccessToken = userIdentity.ExternalId
 	return token, status, nil
 }
+
+func (l *LClient) printMoreThanOneResult(result *ldap.SearchResult) {
+	for idx, e := range result.Entries {
```

Comment:
WHen you move where this function is being called so that it always logs (not just when we get more than one entry), can you also add a loglevel check here so that we dont do this logic when we are not in debug?
something like
if log.LogLevel == log.DebugLevel {
   // do logic
}